### ISSUE TITLE
Move pwntools' shell commands into real pwndbg commands

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#1074]
As described in https://github.com/pwndbg/pwndbg/issues/1068#issuecomment-1218566488, instead of registering Pwntools' shell commands directly, we should:

1. Pull/install the pwntools package as a dependency
2. Create Pwndbg commands for the Pwntools' shell commands - with just better interface/defaults! For example:
* all commands that care about architecture, like `asm` or `disasm`, should have the debugged target architecture (`pwndbg.arch.current`) as the default one
* the `disasm` command should not read from stdin when ran with no arguments as it is now
3. Unregister the shell commands from `shell.py` that are moved to Pwndbg 

With those changes, I think we should also either remove the deprecated shell commands (all the other ones) OR at least remove them from being displayed in the `pwndbg [filter]` output as they do currently -- they basically bloat this display.

Instead of that, if we keep them still (deprecated, until future removal?), we should display them e.g. at the end of the `pwndbg [filter]` display.

**Comment by gsingh93** [comment]
@disconnect3d I don't think this issue will make much progress until we specify specifically what remains to be ported. wdyt?

**Comment by disconnect3d** [comment]
@k4lizen did we remove all shell commands for now? if so, maybe we can also just close this since I think we won't do the move for now?

**Comment by k4lizen** [comment]
yeah i think its fine to close this
